astropy/units/function/core.py

```diff
@@ -571,23 +571,35 @@ def __array_ufunc__(self, function, method, *inputs, **kwargs):
         return super().__array_ufunc__(function, method, *inputs, **kwargs)
 
     # ↓↓↓ methods overridden to change behavior
+    def _maybe_new_view(self, result):
```

Comment:
Shouldn't the new function be defined above the comment?

Reply:
Yes, indeed. Will move it.